lib/pow/store/backend/mnesia_cache.ex

```diff
@@ -184,7 +184,7 @@ defmodule Pow.Store.Backend.MnesiaCache do
         invalidators
 
       {_value, expire} ->
-        case Enum.max([expire - timestamp(), 0]) do
+        case Kernel.max(expire - timestamp(), 0) do
```

Comment:
I wonder, if there is a technical reason for `Enum.max([expire - timestamp(), 0])`, I would assume, that this would generate a little bit more memory usage, and run bit slower.

Reply:
This is better yes!